Application/src/Components/MyPoster.css

```diff
@@ -111,4 +108,4 @@
     display: flex;
     justify-content: center;
     margin: 1em;
-}
\ No newline at end of file
+}

```

Comment:
<img width="1622" alt="CleanShot 2022-07-18 at 16 47 09@2x" src="https://user-images.githubusercontent.com/66916746/179623186-7d94780d-dd0d-4cdb-9745-ac6048c450fe.png">

Reply:
Does the poster card change depending on the tallest poster giving or have you fixed that bug?

Reply:
Every image will be fixed to that size at the moment.

Reply:
To the square aspect ratio?

Reply:
Yeah, I don't know CSS all that well because I never use it.

Reply:
You're good